tests/codebuild/buildspec_china/canary-china.buildspec.yaml

```diff
@@ -0,0 +1,11 @@
+version: 0.2
+phases:
+  build:
+    commands:
+      - cp release/rolebased/china/installer_china.yaml tests/codebuild/
+      - cd tests
+      - aws ecr get-login-password --region $CLUSTER_REGION | docker login --username AWS --password-stdin $ECR_CACHE_URI || true
+      - docker pull ${ECR_CACHE_URI}:latest || true
+      - docker build -f images/Dockerfile.canary.china . -t ${ECR_CACHE_URI}:latest --quiet || true
```

Comment:
If the build fails it will use the cached image 
which might use older versions of our dependencies but that should be okay 
(btw the build fails around 20% of the time as far as i have observed)

i'm injecting our latest code during docker run using "--mount"
so the code will be always the latest one from github even if we use cached image